widget/entry.go

```diff
@@ -487,6 +488,44 @@ func (e *Entry) SetText(text string) {
 	e.propertyLock.Unlock()
 }
 
+// Undo un-does the last modifying user-action.
+func (e *Entry) Undo() {
```

Comment:
The new public methods must also include in the doc:
```
\\
\\ Since: 2.5
```

Reply:
Done.

Reply:
This is not resolved, you missed the ":".

Also, why did you move the public methods back down to the private area? They should be ordered alphabetically with public appearing above private...

Reply:
Okay, I hope it's all correct now. I was a bit tired yesterday.